lib/jellyfish/component/recording.ex

```diff
@@ -25,7 +25,12 @@ defmodule Jellyfish.Component.Recording do
     with {:ok, serialized_opts} <- serialize_options(options, Options.schema()),
          {:ok, credentials} <- get_credentials(serialized_opts, sink_config),
          {:ok, path_prefix} <- get_path_prefix(serialized_opts, sink_config) do
-      output_dir = get_base_path()
+      datetime = DateTime.utc_now() |> to_string()
+      path_suffix = Path.join(options.room_id, "part_#{datetime}")
```

Comment:
in s3 storage recording_id is added so we have to decide where to do it
Code from s3_storage:
```elixir
    path = Path.join([path_prefix, config.recording_id, config.filename])
```